jenkins-build-pack-api/Jenkinsfile

```diff
@@ -789,7 +789,7 @@ def loadServiceInfo(service, domain, serviceUrl, endpointKey) {
 
 	setCredentials(g_base_url + '/platform/login')
 
-	def url = "$serviceUrl?domain=$domain&service=$service"
+	def url = "$serviceUrl?domain=$domain&service=$service&allServices=true"
 
```

Comment:
Same as the above two comments:
1. We need to update this in updateServiceInDB call too.
1. name this parameter as isAdmin